sklearn/tests/test_multiclass.py

```diff
@@ -120,12 +125,12 @@ def test_ovr_partial_fit_exceptions():
     X = np.abs(np.random.randn(14, 2))
     y = [1, 1, 1, 1, 2, 3, 3, 0, 0, 2, 3, 1, 2, 3]
     ovr.partial_fit(X[:7], y[:7], np.unique(y))
-    # A new class value which was not in the first call of partial_fit
-    # It should raise ValueError
+    # If a new class that was not in the first call of partial fit is seen
+    # It should raise Value Error
```

Comment:
```suggestion
    # it should raise ValueError
```

ValueError is the name of the exception. Let's not split it otherwise the casing is inconsistent.